packages/cozy-harvest-lib/src/components/cards/WebsiteLinkCard.spec.jsx

```diff
@@ -1,28 +0,0 @@
-import React from 'react'
-import { render } from '@testing-library/react'
-import I18n from 'cozy-ui/transpiled/react/I18n'
-
-import WebsiteLinkCard from './WebsiteLinkCard'
-import enLocale from '../../locales/en'
-
-describe('WebsiteLinkCard', () => {
-  it('should render', () => {
-    const link = 'https://fc.assure.ameli.fr/FRCO-app/login'
-    const root = render(
-      <I18n dictRequire={() => enLocale} lang="en">
-        <WebsiteLinkCard link={link} />)
-      </I18n>
-    )
-    const linkNode = root.getByText('fc.assure.ameli.fr')
-    expect(linkNode.getAttribute('href')).toEqual(link)
-  })
-  it('should ignore invalid urls', () => {
-    const link = 'www.trainline.fr'
-    const root = render(
-      <I18n dictRequire={() => enLocale} lang="en">
-        <WebsiteLinkCard link={link} />
-      </I18n>
-    )
-    expect(root.queryByText('www.trainline.fr')).toBeFalsy()
-  })
-})
```

Comment:
Why did you remove those tests? There are still relevant. 

Also, I'm not a big fan of the renaming of `WebsiteLinkCard` to `InformationsCards`. In the future, there will a a lot of "InformationsCards" and those informations cards will contain `websitelinkcard` `supportlinkcard` and else.

Reply:
Well, they were linked to a component that was deleted. And replaced by `InformationsCard` indeed. I didn't really see a meaningful way to test the component as it virtually contains no logic. But I should add back test the link fallback

Reply:
About the other part of your comment, the InformationCard is intended to hold all future info displays like support etc. Is that ok for you? That info will just be props @Crash--